AUTHORS

```diff
@@ -24,7 +24,7 @@ affinity <affy at users dot sourceforge dot net>
 akash <akash at users dot sourceforge dot net>
   battery_percent and battery_bar
 
-Asbj�rn Zweidorff Kj�r <bunjiboys at users dot sourceforge dot net>
+Asbjørn Zweidorff Kjær <bunjiboys at users dot sourceforge dot net>
```

Comment:
Revert all these changes.

Reply:
Thanks!

Reply:
Thanks! I've reverted those changes in my latest commit.

Reply:
Thanks! I've reverted them.